invenio.cfg

```
@@ -117,7 +117,8 @@ RDM_COMMUNITIES_ROUTES["community-home"] = (
     communities_home,
 )
 
-ZENODO_ENV = os.environ.get("ZENODO_ENV", "production").lower()
+ZENODO_ENV = os.environ.get("ZENODO_ENV", "local").lower()
+IS_DEPLOYED = ZENODO_ENV in ["production", "qa", "dev"]
```

```suggestion
IS_LOCAL_DEV = ZENODO_ENV == "local"
```

The reasoning is that if e.g. one makes a mistake and sets `ZENODO_ENV=prod` on production, we would still get "some" sort of production behavior (i.e. tasks getting distributed to queues), since `IS_LOCAL_DEV` would be `False`.

---

from what you say I feel like it's more important that line 120 has default = `production`? 

Additionally I don't think `IS_LOCAL_DEV` would ever be true in this case

---

For local development, nobody sets `ZENODO_ENV` in their environment (and I wouldn't start enforcing anything like that). If we have `ZENODO_ENV = os.environ.get("ZENODO_ENV", "local").lower()`, then by default this would be set to `"local"` and thus `IS_LOCAL_DEV` would be set to `True`

---

okay yeah I get it, I just mean that maybe we care about if the ENV isn't set on prod and therefore we shouldn't change the default

---

anyway I've pushed the above

---

Agree regarding being defensive, in which case we should also add something like:

```python
VALID_ENVS = ["local", "dev", "qa", "production"]
if ZENODO_ENV not in VALID_ENVS:
    raise ValueError(f"ZENODO_ENV is set to an invalid value: {ZENODO_ENV}\nExpected one of {VALID_ENVS}"
``` 

I was unhappy with `IS_DEPLOYED = ZENODO_ENV in ["production", "qa", "dev"]`, because in case of a typo, you end up with `IS_DEPLOYED = False` in `prod`